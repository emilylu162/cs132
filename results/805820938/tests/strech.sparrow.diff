1,2c1,4
< The Sparrow-V program is invalid
< Error: The program has no function declarations
---
> 100
> 40464
> 80770
> 121302
