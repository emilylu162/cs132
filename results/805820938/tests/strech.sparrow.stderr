WARNING: A command line option has enabled the Security Manager
WARNING: The Security Manager is deprecated and will be removed in a future release
Error: Cannot invoke "String.endsWith(String)" because "<local8>" is null
java.lang.NullPointerException: Cannot invoke "String.endsWith(String)" because "<local8>" is null
	at ArgumentShuffler.shuffle(ArgumentShuffler.java:29)
	at TranslateVisitor.visit(TranslateVisitor.java:397)
	at TranslateVisitor.visit(TranslateVisitor.java:17)
	at sparrow.Call.accept(Call.java:31)
	at TranslateVisitor.visit(TranslateVisitor.java:99)
	at TranslateVisitor.visit(TranslateVisitor.java:17)
	at sparrow.Block.accept(Block.java:33)
	at TranslateVisitor.visit(TranslateVisitor.java:75)
	at TranslateVisitor.visit(TranslateVisitor.java:17)
	at sparrow.FunctionDecl.accept(FunctionDecl.java:38)
	at TranslateVisitor.lambda$visit$0(TranslateVisitor.java:23)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at TranslateVisitor.visit(TranslateVisitor.java:22)
	at TranslateVisitor.visit(TranslateVisitor.java:17)
	at sparrow.Program.accept(Program.java:28)
	at S2SV.main(S2SV.java:59)
